games/o.yaml

```diff
@@ -1800,6 +1800,36 @@
   url: https://openxcom.org/
   updated: 2020-04-09
 
+- name: OpenXRay
+  originals:
+  - S.T.A.L.K.E.R.
+  type: remake
+  repo: 'https://github.com/OpenXRay/xray-16'
+  development: very active
+  status: playable
+  multiplayer:
+  - Online
+  - LAN
+  content: commercial
+  lang:
+  - C++
+  - Lua
+  framework:
+  - OpenXRay
+  - X-Ray
+  - SDL2
+  - DirectX
+  - OpenGL
+  - Open Dynamics Engine
+  license:
+  - BSD
+  info: 'Crossplatform implementation of the S.T.A.L.K.E.R. games - Windows, linux etc.'
+  images:
+  - 'https://github.com/OpenXRay/xray-16/blob/xd_dev/OpenXRayCover.png'
```

Comment:
please include in-game screenshots to help visitors understand what this game looks like
also, this link should point to the raw image